app/helpers/job_applications_helper.rb

```diff
@@ -83,11 +83,11 @@ def job_application_review_edit_section_text(job_application, step)
   def job_application_review_section_tag(job_application, step)
     form_class = "jobseekers/job_application/#{step}_form".camelize.constantize
```

Comment:
Would it be simpler to pass in the form class directly (in both the helper methods) rather than reconstruct it this way?

Reply:
Makes sense